.travis.yml

```diff
@@ -18,6 +18,9 @@ before_script:
   - export MHV_HOST='https://mock-prescriptions-api.herokuapp.com'
   - export MHV_APP_TOKEN='fake-app-token'
   - export MHV_USER_ID='1435525'
+  - export MHV_APP_TOKEN='fake-app-token'
```

Comment:
good catch - I must have duplicated it in copying and amending it for healthcare api.